src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1687,3 +1687,23 @@ Get_fulltext=
 Download_from_URL=
 
 Decryption_not_supported.=
+
+never=
+every_start=
+check_for_updates=
+download_update=
+reset_ignored_update=
+update=
+new_version_available=
+last_checked=
+ignore_following_update=
+current_version=
+remind_me_later=
+ignore_update=
+weekly=
+monthly=
+couldnt_connect_to_the_updateserver=
```

Comment:
A ' sign seems to be missing here. Also, an underscore/space is missing between update and server